pkg/controller/plan/kubevirt.go

```diff
@@ -1336,23 +1337,19 @@ func (r *KubeVirt) podVolumeMounts(vmVolumes []cnv.Volume, configMap *core.Confi
 
 	switch r.Source.Provider.Type() {
 	case api.Ova:
-		server := r.Source.Provider.Spec.URL
-		splitted := strings.Split(server, ":")
-
-		if len(splitted) != 2 {
-			r.Log.Info("The NFS server path format is wrong")
-			return
+		pvcName := fmt.Sprintf("ova-server-pvc-%s-%s", r.Source.Provider.Name, r.Plan.Name)
+		// If the provider runs in a different namespace then the destination VM,
+		// we need to create the PV and PVC to access the NFS.
+		if r.Plan.Spec.TargetNamespace != r.Source.Provider.Namespace {
```

Comment:
why should it be only on different namespace? shouldn't we always create the PV and PVC?

Reply:
what's done is - if it's the same namespace we don't need to recreate the PV/PVC. we can reuse the existing ones of the OVA server. @bkhizgiy please keep me honest here

Reply:
yes, that's clear but we don't allocate space here - so there's no reason to have such an optimization, it just adds another flow that we need to test - there's no need for that